SQL-Query/Query-VI./SQLAnswer.md

```diff
@@ -0,0 +1,23 @@
+**6. Fetch all the physical items completed from Warehouse in September of 2023.**
+
+```sql
+select 
+  oi.order_id, 
+  oi.order_item_seq_id 
+from 
+  order_item oi 
+  join order_header oh on oh.order_id = oi.order_id 
+  join product p on oi.product_id = p.product_id 
+  join product_type pt on p.product_type_id = pt.product_type_id 
+  join facility f on f.facility_id = oh.origin_facility_id 
```

Comment:
I think we should not have a check on the origin_facility_id on Order Header.
There can be another way to do this. Please check this.

Reply:
Updated query